modules/core/src/main/scala/org/tessellation/infrastructure/snapshot/GlobalSnapshotConsensusFunctions.scala

```diff
@@ -87,10 +86,7 @@ object GlobalSnapshotConsensusFunctions {
       for {
         lastGSHash <- lastGS.value.hashF
         currentOrdinal = lastGS.ordinal.next
-        epochProgress = trigger match {
-          case EventTrigger => lastGS.epochProgress
-          case TimeTrigger  => lastGS.epochProgress.next
-        }
+        currentEpochProgress = lastGS.epochProgress.next
```

Comment:
I think we should advance epoch only on TimeTrigger

Reply:
Ah right. I reverted that change by accident